_src/app/_MultipleWidgetsWithAddBtnMixin.js

```diff
@@ -48,16 +48,21 @@ define([
 
             this.addAddBtnWidget();
         },
-        addAddBtnWidget: function () {
+        addAddBtnWidget: function (id) {
             // summary:
             //      Fires when the add button is pressed on the _addBtn widget
             console.log('app/_MultipleWidgetsWithAddBtnMixin:addAddBtnWidget', arguments);
 
+            if (id === null || id === undefined) {
+                id = this.equipmentCounter;
```

Comment:
`equipmentCounter` should be changed to something more generic since this mixin is used by more than the equipment tab. Something like `itemCounter`?